### PR TITLE
UPDATE: Bias build scripts merge

### PR DESCRIPTION
**Pull request by AlibekMamyrbekov**
1. Changed bias excel file processing
2. Merged several scripts
3. Temporarily removed chembl bias script

### CODE REVIEW

**Comment by AJK-dev** [comment]
Great to see that you've addressed all the Codacy stuff!
Could you do two more things: 
- Remove the 20 ligand model migrations and create one clean one?
- Re-enable the caching of the ligand statistics page?

**Comment by AlibekMamyrbekov** [comment]
> Great to see that you've addressed all the Codacy stuff!
> Could you do two more things:
> 
> * Remove the 20 ligand model migrations and create one clean one? - done
> * Re-enable the caching of the ligand statistics page? - done

File: tools/management/commands/upload_excel_bias.py [hunk 1]
```diff
@@ -46,8 +44,8 @@ class Command(BaseBuild):
 
     help = 'Reads bias data and imports it'
     # source file directory
-    # structure_data_dir = os.sep.join([settings.EXCEL_DATA, 'ligand_data', 'bias'])
-    structure_data_dir = 'excel/'
+    # links_data_dir = os.sep.join([settings.DATA_DIR, 'ligand_data', 'assay_data'])
+    structure_data_dir = os.sep.join([settings.DATA_DIR, 'ligand_data', 'bias_data','excel'])
```
Comment (esguerra) [inline tools/management/commands/upload_excel_bias.py:48]: Remember to put that data in the gpcrdb_data repo and make a pull request.

**Review by esguerra — CHANGES_REQUESTED** [review]
Some minor issues like missing `</a>` and unclosed }

File: ligand/templates/target_details_compact.html [hunk 2]
```diff
@@ -7,309 +7,543 @@
...
+		-webkit-transform: scale(2.5);
+		transform: scale(2.5);
+		background-color: white;
+		border: solid 1px;
+		transform: scale(2.5);
+	}
+
+
+	/* (150% zoom - Note: if the zoom is too large, it will go outside of the viewport) */
+	}
```
Comment (esguerra) [inline ligand/templates/target_details_compact.html:152]: } shouldn't be there.

**Review by esguerra — COMMENTED** [review]
still one more running } at taget_details_compact.html

File: ligand/templates/ligand_info.html [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+			font-size: 100%;
+			font-weight: bold;
+		}
+
+		.highlight {
+			background-color: rgb(204, 229, 255);
+		}
+
+		.name {
+			width: 70%px;
```
[the remark this answers is not in the document]
Comment (AlibekMamyrbekov) [reply ligand/templates/ligand_info.html:462]: LoL, what a coincidence of both in same  place

File: ligand/models.py [hunk 4]
```diff
@@ -11,6 +11,7 @@
 import json
```
[the remark this answers is not in the document]
Comment (AlibekMamyrbekov) [reply ligand/models.py:11]: I am not sure about it, as this part was made by Christian

**Review by AlibekMamyrbekov — COMMENTED** [review]
I am not sure about "quote" in ligand models as i was not using it.  
Html files will be submitted in separate pull requests with corresponding changes in views and model

**Comment by AJK-dev** [comment]
I deleted the dev branch to start fresh from master, but thereby also closed your PR. Oops! 😨

**Comment by esguerra** [comment]
> I deleted the dev branch to start fresh from master, but thereby also closed your PR. Oops! 😨

I think this merge request might not be ready yet.

**Comment by esguerra** [comment]
I have reopened this pull request but think that we should discuss closing it and splitting it into separate pull requests.
This one is mixing changes in the ligand pages and biased signaling pages, making it difficult to separate possible issues if they all come into one pull request.
In my local deployment the selection capability in the ligand pages is lost. Such issues would be easier to track down if split into separate pull requests, as stated before.

**Comment by AlibekMamyrbekov** [comment]
separate into several prs

**Pull request closed without merging**